packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -102,7 +102,6 @@ class _CupertinoTextFieldSelectionGestureDetectorBuilder extends TextSelectionGe
 
   @override
   void onSingleTapUp(TapUpDetails details) {
-    editableText.hideToolbar();
```

Comment:
It's sketchy that we had this kind of behavioral stuff tied to these widgets and not all in one place in EditableText or something.  I'm glad we could remove this line with this PR.  I'm looking forward to your tap gestures project!